ow_faults_detection/src/FaultDetector.cpp

```diff
@@ -0,0 +1,137 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "ow_faults_detection/FaultDetector.h"
+#include <algorithm>
+
+using namespace std;
```

Comment:
Okay, though for a namespace as big and general as 'std', more specific 'using' clauses are better.